src/TimeshiftBuffer.cpp

```diff
@@ -48,7 +55,7 @@ TimeshiftBuffer::~TimeshiftBuffer(void)
   }
   if (m_filebufferReadHandle.IsOpen())
     m_filebufferReadHandle.Close();
-  SAFE_DELETE(m_strReader);
+  SafeDelete(m_strReader);
```

Comment:
Not sure if a `SafeDelete` really needed on destructor, as related class becomes also deleted and `delete` itself also a `NULL` check have.

Reply:
I don't see where the related class is deleted.

Reply:
Has mean to make a easy `delete m_strReader` here :smile: and to leave the `template<typename T> void SafeDelete(T*& p)` there away.

Reply:
Ok, coo!